3-JSS/src/webpackTodoList.js

```diff
@@ -0,0 +1,148 @@
+                break;
+            case 'edit':
+                iconClassName = "fas fa-edit";
+                iconId = `icon-edit-${id}`
+                break;
+            case 'check':
+                iconClassName = "fa-check";
+                iconId = `icon-check-${id}`
+            default:
+        }
+        icon.className = iconClassName;
+        icon.id = iconId
+        return icon
+    }
+    
+    
+    function handleButtonClick(type, todoId) {
+        switch(type) {
+            case 'delete':
+                handleDeleteTodo(todoId)
+                break;
+            case 'edit':
+                handleEditTodo(todoId)
+                break;
+            default:
+                
+        }
+    }   
+    
+    function handleCheckedClick(todoId) {
+        const todo = document.getElementById(`todo-item-${todoId}`)
+        todo.classList.toggle('completed')
+        const isChecked = document.getElementById('checked').checked
+        const myTodo = getTodoById(todoId)
+        myTodo.checked = isChecked
+        updateTodoInLS(myTodo)        
+    }
+    
+    function handleSubmitTodo(e) {
+        e.preventDefault();
+        let input = document.getElementById('new-task-input');
+        if (input.value.trim() != '') { 
+            addTodo(input.value)
+        }else {
+            alert("You didnt enter any Todo")
+        }
+        input.value = '';
+    }
+    
+    function handleDeleteTodo(todoId) {
+        let li = document.getElementById(`todo-list-item-${todoId}`)
+        removeTodoFromLocalStorage(todoId)
+        li.remove()
+    }
+    
+    function handleEditTodo(todoId){
+        let todoItem = document.getElementById(`todo-item-${todoId}`)
+        const myTodo = getTodoById(todoId)
+        myTodo.todoText = todoItem.value
+        updateTodoInLS(myTodo);
+        todoItem.disabled = !todoItem.disabled;
+    }
+    
+    function createTodoItem(todo) {
+        const {todoText, todoId} = todo; 
+        let li = document.createElement('li')
+        let todoDiv = document.createElement('input');
+        todoDiv.className = classes.todoItem
+        todoDiv.classList.add('todo-item');
+        todoDiv.disabled = true
+        todoDiv.id = (`todo-item-${todoId}`)
+        todoDiv.value = todoText
+        todo.checked ? todoDiv.classList.toggle('completed') : ''
+        const deleteButton = createButton('delete', todoId)
+        const editButton = createButton('edit', todoId)
+        const checked = createCheck(todo);
+    
+        li.appendChild(checked)
+        li.appendChild(todoDiv)
+        li.appendChild(deleteButton)
+        li.appendChild(editButton)
+        li.className = classes.todoListItem
+        // li.classList.add('todo-list-item')
+        li.id = (`todo-list-item-${todoId}`)
+        return li
+    }
+    
+    function addTodo(todoText) {
+        const todoListArr = loadTodoListFromLS()
+        const todoId = todoListArr.length
```

Comment:
you have a bug with this logic.
steps to reproduce:
1. create 3 todos
2. delete the middle one
3. create another one

AR: (actual result) - both items will have the same id (`2`)
ER: (expected result) - each todo should have a unique id


in general it's not a good idea to have incremental ids.. it's a source for sync problems and data corruption. it's better to use UUID (read about it)